rust/src/lib/net_state.rs

```diff
@@ -124,6 +124,8 @@ pub struct NetworkState {
     pub(crate) running_config_only: bool,
     #[serde(skip)]
     pub(crate) memory_only: bool,
+    #[serde(skip)]
+    pub(crate) override_iface: bool,
```

Comment:
This is not super important, but I find a bit weird to set this in the NetworkState struct. It represents the current state of the network, so this option has no meaning there. It looks like more correct for MergedNetworkState, and having an argument `override_iface` in `NetworkState::apply`, where it does make sense.

As I say, this is not a must, though.

Reply:
Currently, the `apply()` function does not accept `NetworkStateApplyOption` due to my bad design, so user have to store the option via `NetworkState.set_xxx()`.

Will consider this is we are doing V2 for this.